api/XRView.json

```
@@ -103,13 +103,16 @@
           "spec_url": "https://immersive-web.github.io/webxr-ar-module/#dom-xrview-isfirstpersonobserver",
           "support": {
             "chrome": {
-              "version_added": "86"
+              "version_added": "86",
+              "notes": "Always returns false since no headset with first-person view is supported."
```

I think no partial is correct. I think our guidelines say that a lack of underlying hardware isn't, in and of itself, a trigger of partial implementation. As long as the API does what's expected in the absence of that hardware, then I think we're good.

The note is a good call though.

A small style suggestion too:

```suggestion
              "notes": "Always returns <code>false</code> since no headset with first-person view is supported."
```